### PR TITLE
[NT-1682] Update instances of Kickstarter logo

### PR DESCRIPTION
**Pull request by singhhari**
# 📲 What

We are in the process of replacing our old color scheme with a new palette. We're implementing the changes to the various instances of the Kickstarter logo within the app to reflect these updates.

# 🤔 Why

Most of the changes here are specifically updating instances of a particular shade of green we are currently using. We've consolidated our greens to a set of new ones and will be applying them over the course of a few PRs.

# 🛠 How

- Updated assets.
- Apply tint color for each of these instances.

# 👀 See

Simulator Screen Shot - iPhone 11 - 2020-12-03 at 11 08 14

# ✅ Acceptance criteria

All instances of the Kickstarter logo in the app are a PDF file with the color controlled using code, to make future updates easier

- [ ] Discovery screen
- [ ] Tab bar
- [ ] Sign up / Login screen
- [ ] All instances of the Kickstarter logo in the app have been updated to KDS-create-500

# ⏰ TODO

- [ ] Merge after PR that adds KDS colors and update.

### CODE REVIEW

**Comment by nativeksr** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: Kickstarter-iOS/Views/Cells/DiscoveryOnboardingCell.swift [hunk 1]
```diff
@@ -62,6 +62,7 @@ private let discoveryOnboardingTitleStyle: LabelStyle = { label in
 private let discoveryOnboardingLogoStyle: ImageViewStyle = { imageView in
   imageView
     |> \.contentMode .~ .scaleAspectFit
+    |> \.tintColor .~ .ksr_green_500
```
Comment (justinswart) [inline Kickstarter-iOS/Views/Cells/DiscoveryOnboardingCell.swift:65]: Once rebased this can change to `ksr_create_700`.

**Review by justinswart — CHANGES_REQUESTED** [review]
We'll need to rebase/merge this on `NT-1090-kds-updates-feature`. We're using a feature branch for this so that it goes out separately from the next release. That branch also has some of the colour changes already so the particular green that you need is available there.

File: Kickstarter-iOS/Views/Cells/DiscoveryOnboardingCell.swift (hunk 1 above)
Comment (singhhari) [reply Kickstarter-iOS/Views/Cells/DiscoveryOnboardingCell.swift]: Merged the changes from your work in `NT-1090` and updated some snapshots after resolving merge conflicts. Is there an existing feature branch to change the base of this PR?

**Pull request closed without merging**